relay-event-normalization/src/normalize/span/description/sql/mod.rs

```diff
@@ -672,6 +672,30 @@ mod tests {
         "SELECT * FROM a WHERE b = %s AND (obj -> %s <@ %s) AND b <> %s"
     );
 
+    scrub_sql_test!(
+        digits_in_table_name,
+        "SELECT * FROM temp_12c",
+        "SELECT * FROM temp_00c"
+    );
+
+    scrub_sql_test!(
+        single_digit_in_table_name,
+        "SELECT * FROM temp_1c",
+        "SELECT * FROM temp_1c"
+    );
+
+    scrub_sql_test!(
+        digits_in_compound_table_name,
+        r#"SELECT * FROM "foo"."temp_12c""#,
+        "SELECT * FROM temp_00c"
+    );
+
+    scrub_sql_test!(
+        uuid_in_table_name,
+        "SELECT * FROM prefix_0123456789abcdef0123456789ABCDEF_006_suffix",
+        "SELECT * FROM prefix_00_00_suffix"
```

Comment:
@gggritso let me know what you think of the placeholder here. `00` is not very explicit, but I like that it's short and still vaguely implies that some generated number was here. I'm open to using a different placeholder though.

Reply:
IMO any of the known placeholder strings would be great here! `"%s` is my favourite, since it's so familiar, but even `"?"` or others

Reply:
Makes sense!